src/commands/upload_file.ts

```diff
@@ -86,7 +92,10 @@ new CLICommand({
 
 			const wallet = readJWKFile(options.walletFile);
 			const priceEstimator: ARDataPriceEstimator = (() => {
-				if (filesToUpload.length > ARDataPriceRegressionEstimator.sampleByteVolumes.length) {
+				if (
+					filesToUpload.length > ARDataPriceRegressionEstimator.sampleByteVolumes.length ||
```

Comment:
Now that we use the estimator for settings rewards on tips and metadata as well, we can always use it when there's at least 1 file since 1 file is 3 requests for pricing data anyway (file, meta, tip).